snuba/migrations/snuba_migrations/outcomes/0002_outcomes_remove_size_and_bytes.py

```diff
@@ -0,0 +1,30 @@
+from typing import Sequence
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+
+
+class Migration(migration.MultiStepMigration):
+    """
+    We added the size and bytes_received columns on 15 Dec 2019 and reverted the next
+    day. This migration ensures the column is dropped for all users on the off chance
+    they had upgraded or attempted to upgrade to a version released in that window.
+    """
+
+    blocking = False
+
+    def forwards_local(self) -> Sequence[operations.Operation]:
+        return [
+            operations.DropColumn(StorageSetKey.OUTCOMES, "outcomes_raw_local", "size"),
+            operations.DropColumn(
+                StorageSetKey.OUTCOMES, "outcomes_hourly_local", "bytes_received"
+            ),
```

Comment:
Does this work because the column is there in the destination table of the materialized view but not in the schema of the materialzied view (or in the materialization statement) itself ?

Reply:
Yes - since it's not possible to add a column to an existing materialized view (need to drop and recreate),  the original DDL never did that.